docs/index.rst

```diff
@@ -12,26 +12,16 @@ StarCraft
    reference/index
    explanation/index
 
-.. grid:: 1 1 2 2
-
-   .. grid-item-card:: :ref:`Tutorial <tutorial>`
-
-      **Get started** with a hands-on introduction to Starcraft
-
-   .. grid-item-card:: :ref:`How-to guides <howto>`
-
-      **Step-by-step guides** covering key operations and common tasks
-
-.. grid:: 1 1 2 2
-   :reverse:
-
-   .. grid-item-card:: :ref:`Reference <reference>`
-
-      **Technical information** about Starcraft
-
-   .. grid-item-card:: :ref:`Explanation <explanation>`
-
-      **Discussion and clarification** of key topics
+.. list-table::
```

Comment:
how come this was changed?

Reply:
Grid items are provided by the sphinx design extension which is not in the mininal install, so the build will fail if these roles are included in the test set.

Reply:
But isn't the canonical-sphinx documentation built with [the full set](https://github.com/canonical/canonical-sphinx/pull/50/files#diff-03efc769b870804394632e45d7885272b44c16939517fb31c9d7c614d2ffae57L27)?

Reply:
I added a tox test for the minimal set which fails if this change isn't made.

We could add another set of docs to test against that includes all the functionality provided by the optional extensions - but a lot of the optional functions are tested downstream anyway.

Reply:
alright, but moving forward I think we should be careful with changing these docs for the sake of testing. I know they're pretty bare-bones now but ideally they'll document the extension itself, so chances are they won't be build-able with the minimum set of dependencies.

But that can be for the future!

Reply:
I don't think this extension will require a dedicated documentation site to provide information on the extension. It is not complex enough to require more than a detailed readme.